src/components/LoginPage/Slide3.js

```diff
@@ -0,0 +1,164 @@
+  width: 1.5%;
+  height: auto;
+  transform: rotate(270deg);
+  position: absolute;
+  left: 36%;
+  top: 38%;
+`;
+const ArrowRight = styled.img`
+  width: 1.5%;
+  height: auto;
+  transform: rotate(90deg);
+  position: absolute;
+  right: 36%;
+  top: 38%;
+`;
+const Hint = styled.div`
+  position: relative;
+  width: 34%;
+  background: #0E1736;
+  border-radius: 5px;
+  box-shadow: inset 0px 4px 2px rgba(0, 0, 0, 0.25);
+  padding-left: 2.7%;
+  padding-right: 2.7%;
+  padding-top: 4%;
+  padding-bottom: 4%;
+  color: #E8E9F8;
+  text-transform: uppercase;
+  font-weight: 700;
+  margin-top: 1vw;
+  margin-bottom: 1vw;
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+const CaptionTextSlide3 = ({x = 0, y = 15, fill = "#FFFFFF", fontWeight = 700, text}) => {
+  return <text x={x} y={y} fill={fill} font-weight={fontWeight}>{text}</text>;
+};
+
+const CaptionBoxSlide3 = ({viewBoxWidth = 100, viewBoxHeight = 21, ...rest}) => {
+  return <Caption viewBox={`0 0 ${viewBoxWidth} ${viewBoxHeight}`} {...rest} />;
+};
+
+
+
+export default class Slide3 extends Component {
+    render() {
+        return (
+            <Slide>
+            <H2><b>Edit Notes Content</b>
+              <br />
+              And Attach Files
+            </H2>
+            <Figure>
+              <Img className='note-example' src={imageNoteContent} shadow />
+            </Figure>
+            <Figure>
+              <ArrowUp src={arrow} />
+              <Hint>
+                <CaptionBoxSlide3 viewBoxWidth="155" viewBoxHeight="19" left="12%" top="27%" width="76%">
+                  <CaptionTextSlide3 y="17" text="Edit Note Content"/>
+                </CaptionBoxSlide3>
+              </Hint>
+              <ArrowDown src={arrow} />
+            </Figure>
+            <Figure className='edit-example'>
+              <Img src={imageMindmapExplanation} shadow />
+                <CaptionBoxSlide3  left="78.5%" top="5%" width="15%">
+                  <CaptionTextSlide3 fill="#96ffff" text="Neural"/>
+                <CaptionTextSlide3 x="47" text="Notes"/>
+                </CaptionBoxSlide3>
+              <CaptionBoxSlide3 viewBoxWidth="80" viewBoxHeight="40" left="43.5%" top="16.5%" width="14%">
+                  <CaptionTextSlide3 y="14" text="VACATION"/>
+                  <CaptionTextSlide3 x="22" y="34" text="2018"/>
+              </CaptionBoxSlide3>
+              <CaptionBoxSlide3 viewBoxWidth="90" viewBoxHeight="44" left="12%" top="28.5%" width="12%">
+                <CaptionTextSlide3 x="22" y="14" text="Hotel"/>
+                  <CaptionTextSlide3 y="34" text="Booking.pdf"/>
+                </CaptionBoxSlide3>
+              <CaptionBoxSlide3 viewBoxWidth="72" viewBoxHeight="44" left="77%" top="28.5%" width="10%">
+                <CaptionTextSlide3 x="4" y="14" text="Airplane"/>
+                <CaptionTextSlide3 y="34" text="ticket.pdf"/>
+              </CaptionBoxSlide3>
+            </Figure>
+            <div className='container-for-hint'>
```

Comment:
Why don't we make it a `HintContainer` component?

Reply:
Done in commit [db8d5d2](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/db8d5d213e208ca30de77f1c6674c40053f31f87)